safe/defaults.py

```diff
@@ -140,6 +140,7 @@ def default_gender_postprocessor():
     gender = BooleanParameter()
     gender.name = 'Gender'
     gender.value = True
+    gender._description = 'Enable a gender specific breakdown in the report'
```

Comment:
Just note that the changes made to defaults.py are going to conflict with my work in #1792